regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -22,9 +22,7 @@ var CommentReviewView = Backbone.View.extend({
     this.docId = this.$el.data('doc-id');
     this.template = _.template($('#comment-template').html());
 
-    this.mode = 'preview';
-    this.previewUrl = null;
-
+    this.previewLoading = false;
```

Comment:
Curious: can you imagine any race-cases between the the user's actions and this state? Would it make sense to pass this as a flag to `render` instead?